lib/profiling/simple_timing/simple_timing.f90

```diff
@@ -205,13 +212,14 @@ subroutine profile_PSyDataShutdown()
     print *
     print *,"==========================================="
     print *, heading, spaces(1:max_len - len(heading)),                       &
```

Comment:
You probably don't want to hear this but the formatting I get for this table is very 'wide':
![image](https://user-images.githubusercontent.com/17409212/94677562-03ed1e00-0315-11eb-916e-2f10cd7dd504.png)
Is that what you intended? It doesn't really matter but I thought I'd flag it in case you care :-)

Reply:
Yes, I don't see a good way around this. The text field width is determined dynamically, the rest is spaces from Fortran formatting (or a tab to align columns). We could use better formatting, but then ... we don't really know the order of magnitude of the values ... we could have 1/100 seconds, or 1000 seconds ... messy to format this. This way it works, and I am happy to leave it this way. Other options would be to remove columns (min, max?), or maybe make this dependent on PSY_DATA_VERBOSITY. If you want it changed, please open an issue :)